src/App.tsx

```diff
@@ -15,71 +16,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selected: string;
+  isPressed: boolean;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selected: 'Jam',
+    isPressed: true,
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  render() {
+    const { selected, isPressed } = this.state;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        <h1 className="title is-flex is-align-items-center">
+          {goods.includes(selected) ? (
+            <>
+              {`${selected} is selected`}
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  this.setState({
+                    selected: 'No goods selected',
+                    isPressed: false,
+                  });
+                }}
+              />
+            </>
+          ) : ('No goods selected')}
+        </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+        <table className="table">
+          <tbody>
+            {goods.map(good => (
+              <tr data-cy="Good" key={good} className={`${(selected === good) && 'has-background-success-light'}`}>
```

Comment:
```suggestion
              <tr
                data-cy="Good"
                key={good}
                className={`${(selected === good) && 'has-background-success-light'}`}
              >
```
don't forget about formatting